docs/source/services/timelock_service/index.rst

```diff
@@ -0,0 +1,38 @@
+.. _external-timelock-service:
+
+External Timelock Service
+=========================
+
+.. warning::
+
+   Running external timestamp and lock services is currently an experimental feature.
+   If you are interested in testing it, please contact the AtlasDB team.
+
+.. danger::
+
+   Improperly configuring one's cluster to use external timestamp and lock services can result in **SEVERE DATA
+   CORRUPTION**! Please contact the AtlasDB team if you wish to try this feature.
```

Comment:
Oops, the ending bit of the previous one should say "if you're configuring the client" - good catch.

You can set up the timelock server and play with it without a risk of severe data corruption - the risky bits start when you actually point a cluster at it, or you run the migration commands :)